packages/shared/src/styles/constants.scss

```diff
@@ -23,6 +23,7 @@ $color-red-2: #cc2e3d;
 $color-red-3: #a32430;
 $color-red-4: #d33636;
 $color-red-5: #eb3e48;
+$color-red-6: #ec3f3f29;
```

Comment:
Is this a valid value? Why does it have 8 characters inside?

Reply:
It translates to rgba(236, 63, 63, 0.16)

_Image from Figma:_
![Screen Shot 2022-04-07 at 9 26 49 AM](https://user-images.githubusercontent.com/98398322/162126550-6b9fe718-34e9-45f6-9d0c-e860c252e60c.png)

Reply:
Where did you get this value? Please get it from Figma, it should have a 6 digit hexadecimal value.

Reply:
Ok, thanks for explanation👍🏻

Reply:
You don't need to add a color for opacity only since #ec3f3f already exists as `$color-red-1`

![image](https://user-images.githubusercontent.com/58209918/162371111-1676e4b4-fe36-4b59-aa65-255a4e86ea0f.png)

Reply:
Will change

Reply:
You don't need to add a new color for this is what I'm saying, you can use the existing color and add opacity.

Reply:
ok. will work on that